cardano-node/src/Cardano/Tracing/Tracers.hs

```diff
@@ -502,19 +502,22 @@ teeForge ::
 teeForge ft nodeKern tverb tr = Tracer $ \ev@(WithSeverity sev event) -> do
   flip traceWith ev $ fanning $ \(WithSeverity _ e) ->
     case e of
-      Consensus.TraceNodeCannotForge {} -> teeForge' (ftTraceNodeCannotForge ft)
-      Consensus.TraceForgeStateUpdateError{} -> teeForge' (ftTraceForgeStateUpdateError ft)
-      Consensus.TraceForgedBlock{} -> teeForge' (ftForged ft)
       Consensus.TraceStartLeadershipCheck{} -> teeForge' (ftForgeAboutToLead ft)
+      Consensus.TraceSlotIsImmutable{} -> teeForge' (ftTraceSlotIsImmutable ft)
+      Consensus.TraceBlockFromFuture{} -> teeForge' (ftTraceBlockFromFuture ft)
+      Consensus.TraceBlockContext{} -> panic "no point in tracing this"
```

Comment:
@denisshevchenko I left 6 `panic`s in this file because IMO they're no point in forwarding these (debug) messages. What should we do?

Reply:
panic is probably not the right thing to enter production.
do you want to suppress this particular messages instead?

Reply:
@mrBliss, this case ought to return a tracer -- so if we want to ignore something, the current solution is by returning a `nullTracer`.

Reply:
Yeah, the `panic` is of course not meant to be merged, it's only to signal that this still needs to be addressed :slightly_smiling_face: 

@deepfire Thanks for the answer!

Reply:
I would really prefer not to have a `panic` there, expecially not with such a poor error message.

Using `nullTracer` seems like the sensible solution.